socialdistribution/profiles/views.py

```diff
@@ -1,8 +1,9 @@
-from django.shortcuts import render, redirect
+from django.shortcuts import render, redirect, render_to_response
```

Comment:
`render_to_response` is deprecated and removed in django 3+ and does not run. Remove import also not used anywhere.